README.md

```diff
@@ -194,21 +194,16 @@ Some connectors can be installed with the formula. See `brew info marvin182/zapf
 __Build via vcpkg:__
 
 You can download and install sqlpp11 using the [vcpkg](https://github.com/Microsoft/vcpkg) dependency manager:
-   
+
 ```bash
 git clone https://github.com/Microsoft/vcpkg.git
 cd vcpkg
 ./bootstrap-vcpkg.sh
 ./vcpkg integrate install
-vcpkg install sqlpp11
+vcpkg install sqlpp11[<database>]
```

Comment:
I have no clue about `vcpkg`, so apologies for the question: Is there maybe a `space` missing here?

i.e. `sqlpp11mysql` vs `sqlpp11 mysql`

Reply:
No. The point is that users shall select an actual database "feature", e.g. `sqlpp11[mysql]`.

I understand that sqlpp11 is header-only, but from a package management perspective, the backends are transitive dependencies of sqlpp11.

Let's use an example instead, and resolve two additional quirks (path to vcpkg; zsh syntax):
```suggestion
./vcpkg install 'sqlpp11[mysql]' # or other database feature
```
Okay?